packages/database/src/migrations/20220218050602-AddCovidKiribatiProject-modifies-data.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+import { insertObject, codeToId, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const PROJECT_NAME = 'COVID-19 Kiribati';
+const PROJECT_CODE = 'covid_kiribati';
+const DASHBOARD_CODE = 'covid_ki';
+const DASHBOARD_NAME = 'COVID-19 Kiribati';
+const MAP_OVERLAY_GROUP_CODE = 'covid_ki';
+const MAP_OVERLAY_GROUP_NAME = 'COVID-19 Kiribati';
+const PROJECT = {
+  code: PROJECT_CODE,
+  description: 'COVID-19 case surveillance and data visualisations in Kiribati',
+  sort_order: 16,
+  image_url:
+    'https://tupaia.s3.ap-southeast-2.amazonaws.com/thumbnails/uploads/covid_kiribati_background.jpeg',
+  default_measure: '126,171',
+  dashboard_group_name: DASHBOARD_NAME,
+  permission_groups: '{COVID-19 Kiribati Admin,COVID-19 Kiribati}',
+  logo_url:
+    'https://tupaia.s3.ap-southeast-2.amazonaws.com/thumbnails/uploads/covid_kiribati_logo.jpeg',
+};
+const ENTITY = {
+  code: PROJECT_CODE,
+  name: PROJECT_NAME,
+  type: 'project',
+};
+
+const getWorldBounds = async db =>
+  db.runSql(`select bounds from entity where code = 'World' limit 1;`);
+
+const addDashboard = async db => {
+  await insertObject(db, 'dashboard', {
+    id: generateId(),
+    code: DASHBOARD_CODE,
+    name: DASHBOARD_NAME,
+    root_entity_code: PROJECT_CODE,
+  });
+};
+
+const addMapOverlayGroup = async db => {
+  await insertObject(db, 'map_overlay_group', {
+    id: generateId(),
+    code: MAP_OVERLAY_GROUP_CODE,
+    name: MAP_OVERLAY_GROUP_NAME,
+  });
+
+  const mapOverlayGroupId = await codeToId(db, 'map_overlay_group', 'Root');
+  const childId = await codeToId(db, 'map_overlay_group', MAP_OVERLAY_GROUP_CODE);
+
+  await insertObject(db, 'map_overlay_group_relation', {
+    id: generateId(),
+    map_overlay_group_id: mapOverlayGroupId,
+    child_id: childId,
+    child_type: 'mapOverlayGroup',
+  });
+};
+
+const addEntity = async db => {
+  const id = generateId();
+  const { bounds } = (await getWorldBounds(db)).rows[0];
+  const parentId = await codeToId(db, 'entity', 'World');
+  await insertObject(db, 'entity', {
+    id,
+    bounds,
```

Comment:
Are you sure you want the bounds to be the same as World, not the same as Kiribati? Or potentially a slightly further zoomed out view than what we have for Kiribati at the moment

Reply:
I was copying some other project creations here, so didn't really understand the usage. I will change that.

Reply:
Oh yep, makes sense! This refers to the geographic bounds that Tupaia should zoom to when you visit that entity. For projects where there are multiple countries involved, it makes sense to use bounds that capture all of those countries, so the World bounds is an easy shortcut (as it's actually just a decent view of the Pacific, despite being called World). For this project though, we likely want a tighter view that's zoomed in on just Kiribati (which is admittedly a very stretched out nation!)